state/oci/objectstorage/objectstorage_test.go

```diff
@@ -0,0 +1,203 @@
+package objectstorage
+
+// for example in ~/dapr-dev/components-contrib
+// go test -v github.com/dapr/components-contrib/state/oci/objectstorage
+// to run the test set for OCI ObjectStorage service based StateStore.
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dapr/components-contrib/state"
+	"github.com/dapr/kit/logger"
+)
+
+var dummyOCIObjectStorageConfiguration = map[string]string{
```

Comment:
Please add more cases for config to ensure any wrong config would return error.

Reply:
I have included test cases for each of the configuration properties - all are mandatory at this point.